frontend/src/pages/login/Login.jsx

```diff
@@ -48,7 +48,8 @@ export default function Login() {
 
   useEffect(() => {
     if (isLoggedIn) {
-      navigate(-1, { replace: true });
+      const next = location.state === null ? '/' : location.state.next
```

Comment:
```suggestion
      const next =!location.state ? '/' : location.state.next;
```
location.state가 undefined이 나올 수 있으므로 이 코드는 어떠신가요?

Reply:
수정했습니다!!